pom.xml

```diff
@@ -55,6 +55,11 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-resteasy-reactive-jackson</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.module</groupId>
+            <artifactId>jackson-module-jsonSchema</artifactId>
+            <version>2.17.1</version>
```

Comment:
@zakkak Could we move the version to `<properties>`, please?
Also, isn't this pulled already by Quarkus...?

Reply:
> @zakkak Could we move the version to `<properties>`, please? 

On it.

> Also, isn't this pulled already by Quarkus...?

Surprisingly, no :)

Reply:
Done